dataflow/src/ops/union.rs

```diff
@@ -9,9 +9,9 @@ enum Emit {
         emit: HashMap<IndexPair, Vec<usize>>,
 
         // generated
-        emit_l: Map<Vec<usize>>,
+        emit_l: BTreeMap<LocalNodeIndex, Vec<usize>>,
```

Comment:
This change was made because there are now a lot of IDs, right? Why a `BTreeMap` over a `HashMap`? I guess we expect there to be relatively few entries?

Reply:
Yeah, this was changed because `Map` has the size of the biggest ID. I believe we discussed `HashMap` vs. `BTreeMap` and ended up deciding that the `BTreeMap` would be better, don't quite remember why though :(